docs/whats-new.asciidoc

```diff
@@ -4,156 +4,149 @@
-
-* You can now check how many other rules are affected by an exception by clicking *Affects _X_ rules* next to an exception list item.
+The DED integration package protects your data from being exfiltrated to external web services (technical preview only).
 
 [role="screenshot"]
-image::whats-new/images/8.5/affects-rule.png[Shows how many rules are affected by an exception]
-
-*New prebuilt rules*
+image::whats-new/images/8.6/DED-integration.png[DED integration]
 
-28 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added in 8.5.0. 
+[float]
+== Entity Analytics dashboard enhancements
 
-[discrete]
-== Endpoint response enhancements 
+The following enhancements have been added to the {security-guide}/detection-entity-dashboard.html[Entity Analytics dashboard] to improve efficiency in responding to emerging threats: 
 
-*Updated messaging in response console*
+* Hover over a host name or user name to display inline actions: *Add to timeline investigation*, which adds the selected value to a new Timeline, and *Copy to Clipboard*, which copies the host name value for you to paste later.
+* Launch Timeline by clicking the number link in the *Alerts* column. 
+* Install and run a {ml} job directly from the *Notable Anomalies* table  by clicking *Run job*. 
 
-The {security-guide}/response-actions.html[response console] UI now displays a message if response action commands aren't supported by an installed version of {agent}, which must be 8.4 or later. 
+[role="screenshot"]
+image::whats-new/images/8.6/entity-dashboard.png[Entity Analytics Dashboard]
 
-*Response actions history log for all endpoints*
+[float]
+== Exceptions enhancements
 
-You can now filter and search endpoint response actions history for an endpoint. In addition, there's now a standalone {security-guide}/response-actions-history.html[Response actions history] page that shows an action log of all endpoints.
+The following enhancements have been added to the exceptions feature to improve the UI and simplify management: 
 
+* The Exceptions lists page has been renamed to *Shared Exception Lists*, where you can create, manage, and import shared exception lists.
+* There are now two types of exception lists:
+** *Default exception lists*: A default rule list is a group of exceptions that belong to a single rule. Other rules cannot use exceptions within a rule’s default rule list.
++
 [role="screenshot"]
-image::whats-new/images/8.5/response-history.png[Response history action log]
+image::whats-new/images/8.6/default-rule-list.png[Default rule list]
++
+** *Shared exception lists*: Shared exception lists allow you to group exceptions together, then apply them to multiple rules. Exception lists created in {stack} versions 8.5 and earlier are saved as exception lists when upgrading to 8.6 or later. You can access all shared exception lists from the *Shared Exception Lists* page (*Manage -> Shared Exception Lists*).
```

Comment:
```suggestion
** *Shared exception lists*: Shared exception lists allow you to group exceptions together, then apply them to multiple rules. Exception lists created in {stack} versions 8.5 and earlier are saved as shared exception lists when upgrading to 8.6 or later. You can access all shared exception lists from the *Shared Exception Lists* page (*Manage -> Shared Exception Lists*).
```
Is this correct? I'm wondering since it's in this paragraph rather than the previous one that describes default exception lists

Reply:
Previous para should have said `default rule list` - so it's correct. Thanks for checking.